client/packages/system/src/Log/Components/ListView.tsx

```diff
@@ -12,7 +12,12 @@ export const LogList: FC<{ recordId: string }> = ({ recordId }) => {
   const { data, isError, isLoading } = useLog.document.listByRecord(recordId);
   const t = useTranslation();
   const columns = useColumns<LogRowFragment>([
-    { key: 'datetime', label: 'label.date', format: ColumnFormat.Date },
+    {
+      key: 'datetime',
+      label: 'label.date',
+      format: ColumnFormat.Date,
+      width: 200,
```

Comment:
the date column was spreading wide, even on the laptop screen. Looks better like this 😉